packages/router/src/bindings/fastify/index.ts

```diff
@@ -47,7 +47,7 @@ export const bindFastify = () =>
           return res.code(401).send("Unauthorized to perform this operation");
         }
         try {
-          const result = await contractWriter.removeLiquidity(
+          const result = await contractWriter.removeLiquidityTransactionManager(
```

Comment:
what if they have a contract based router?

Reply:
Not handled yet, can deal with this later.